setup.py

```diff
@@ -437,7 +437,7 @@ def run(self):
             'sympy>=1.2',
             'numpy>=1.10.4',
             'IPython>=1.0',
-            'unyt>=2.2.2',
+            'unyt>=2.7.2',
```

Comment:
Are our unyt fixes not backwards compatible with previous unyt versions? They have to install this version of unyt for yt to work?

Reply:
I would say that since our answer tests require this version, we should as well -- the main thing I want to make sure of is the `mu` symbol being correct.

Reply:
Yes, I think it's reasonable that we at least match the versions we test in test_minimal_requirements. Hopefully as time moves on the versions between test_requirements and test_minimal_requiements start to differ.